docs/supported_platforms_and_configurations.md

```diff
@@ -76,11 +76,11 @@ For more information see [conan-io/conan-docker-tools](https://github.com/conan-
 
 - Python: 3.7.12
 - CMake: 3.20.1
-- Compilers: Apple-clang versions 11.0.3, 12.0.5, 13.0.0
-- Macos SDK versions (for each apple-clang version respectively): 10.15, 11.3
+- Compilers: Apple-clang versions 13.0.0
```

Comment:
Not accurate IIRC, we are supporting 2 apple-clang versions, the latest and the previous one?